catalog/core/catalog-core-impl/filter-proxy/src/main/java/ddf/catalog/filter/proxy/adapter/GeotoolsFilterAdapterImpl.java

```diff
@@ -319,22 +326,13 @@ public Object visit(PropertyIsEqualTo filter, Object delegate) {
     List<Object> functionArgs = filterValues.functionArgs;
 
     // Special case to handle relative temporal queries
-    if (literal instanceof String
-        && RELATIVE_TEMPORAL_MATCHER.matcher((String) literal).matches()) {
+    if (literal instanceof String && RELATIVE_TEMPORAL_REGEX.matcher((String) literal).matches()) {
```

Comment:
If you have time I would appreciate if we could find a way to do this as a proper function like ProximityFunction so that it is recognized and parsed as such instead of having to do a regex.

Reply:
I added a comment at the regex to steer people away from using regexs and I created a ticket to turn this into a function in the future.